src/igtcloud/client/tools/common.py

```diff
@@ -64,7 +70,7 @@ def filter_by_ext(ext: Optional[str]):
     if ext is None:
         return None
 
-    def ext_filter(file: File, study: Optional[RootStudy] = None) -> bool:
```

Comment:
Why is this removed?

Reply:
The `study` parameter was not being used.